frontend/src/contexts/UserContext.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+
+/**
+ * NOTE:
+ * This is still work in progress.
+ * - get rid of all the "any" typescript declarations
+ * - once auth-backend is known and online, refine the process in this file
+ */
+
+const TOKEN_NAME = "tyb-jwt";
+
+interface UserContextProvider {
+  children: React.ReactChild;
+}
+
+const initialState = {
+  user: {
+    name: ""
+  },
+  hospitalId: "35ad5a65-f00a-4eb6-8302-3f170863b15c",
+  isAuthenticated: true
+};
+
+export const UserContext = React.createContext(initialState);
+
+const reducer = (state: any, action: any): any => {
```

Comment:
As you already use a reducer here, do we want to deploy redux? 🙂

Reply:
i thought about this too! i came to the conclusion that we dont have a lot of "global state" so i think redux would be a bit of a overkill for our needs. but im happy to change my mind if you guys think otherwise?!

Reply:
I am open to more lightweight systems as well 😉 I am just very familiar with redux, that is why I suggested it 😉